README.md

```diff
@@ -242,6 +242,10 @@ During a fresh Nextcloud installation, the latest version (from the image) of th
 The copied files, however, are **not** automatically overwritten whenever you update your environment with a newer Nextcloud image. This is to prevent local changes in `/var/www/html/config` from being unexpectedly overwritten. This may lead to your image-specific configuration files becoming outdated and image functionality not matching that which is documented.
 
 A warning will be generated in the container log output when outdated image-specific configuration files are detected at startup in a running container. When you see this warning, you should manually compare (or copy) the files from `/usr/src/nextcloud/config` to `/var/www/html/config`.
+A command to copy these configs would e.g. be:
+```console
+docker exec <container-name> sh -c "cp /usr/src/nextcloud/config/*.php /var/www/html/config"
```

Comment:
Yeah, if this is what the maintainers think?

That said, the files shipped in the container are likely always newer than any manual config adjustments, if you run this command after an update, because well… you likely adjusted the config before the update, anyway… so hmm?